content/docs/other-guides/usage-reporting.md

```diff
@@ -4,49 +4,82 @@ description = "Opting in or out of anonymous usage data reported from Kubeflow"
 weight = 40
 +++
 
-When enabled, Kubeflow will report **anonymous** usage data using [spartakus](https://github.com/kubernetes-incubator/spartakus), Kubernetes' reporting tool. Spartakus **does not report any personal information**. See [here](https://github.com/kubernetes-incubator/spartakus) for more detail.
-This is entirely voluntary and you can opt out by doing the following:
+When enabled, Kubeflow will report **anonymous** usage data using 
+[Spartakus](https://github.com/kubernetes-incubator/spartakus), Kubernetes' 
+reporting tool. Spartakus does not report any personal information. 
+See the [Spartakus docs](https://github.com/kubernetes-incubator/spartakus) for 
+more detail. 
+
+Allowing usage reporting is entirely voluntary.
+
+{{% alert title="Reporting usage data is one of the most significant contributions you can make to Kubeflow" color="info" %}}
+Please consider allowing the reporting of usage data.
+The data helps the Kubeflow community to improve the project and helps the many 
+companies working on Kubeflow justify continued investment.
+{{% /alert %}}
+
+## Disable usage reporting on an existing Kubeflow deployment
+
+If you've already deployed Kubeflow, run the following command to disable usage 
+reporting on your existing deployment. The command removes the 
+`spartakus-volunteer` component:
```

Comment:
nit: We are adopting the terminology of "application" as opposed to "component"
So suggest changing it to

"the spartakus-volunteer application"

Reply:
Done.